spec/lib/cooked_post_processor_spec.rb

```diff
@@ -1621,7 +1621,6 @@
               <p>This post has an S3 video onebox:</p><div class="onebox video-onebox">
                 <video width="100%" height="100%" controls="">
                   <source src="#{secure_url}">
-                  <a href="#{secure_url}">#{secure_url}</a>
```

Comment:
Circling back to the question from the previous review - do we know why all these `<a href>`s are gone now? 😃

Reply:
Yes this one should've been added back since we're using html5_fragments in the oneboxer now. (Test failed for correct reason)